docs/markdown/Dependencies.md

```diff
@@ -849,6 +849,34 @@ version.
 
 *New in 0.54.0* the `system` method.
 
+## DIA SDK
+
+*(added 1.6.0)*
+
+Microsoft's Debug Interface Access SDK (DIA SDK) is available only on Windows,
+when using msvc, clang-cl or clang compiler from Microsoft Visual Studio.
+
+Version info is not available for this dependency.
```

Comment:
I'm not sure this sentence is so important. I would rather put it at the end of this section.

Reply:
Actually, I forgot to update docs regarding the version handling. Updated now.